electricpy/__init__.py

```diff
@@ -6573,9 +6573,277 @@ def wireresistance(length=None,diameter=None,rho=16.8*10**-9,R=None):
         A = rho*length/R
         return _np.sqrt(4*A/pi)
         
+def ic_555(mode,R=None,C=None,f=None,t_high=None,t_low=None):
+
```

Comment:
This function looks great! I think we'll just need to work on the documentation for it as the docstring. Perhaps this might be a starting point:

```
555 Integrated Circuit Calculator

Evaluate a number of common attributes related to the common 555 integrated
circuit including time period, frequency, duty cycle, time spent low during each
cycle, time spent high during each cycle.

.. math:: <various-math-equations for each output>

.. image:: <a-diagram-showing-the-circuit-configuration-for-the-ic>

Parameters
----------
mode:  ['astable', 'monostable']
            The operating constraint of the 555 timer.
R:  float, optional
      <describe-the-resistor>
C:  float, optional
      <describe-the-capacitor>
f:  float, optional
    Electrical system frequency in Hertz.
t_high:  float, optional
             <describe>
t_low:   float, optional
             <describe>

Returns
-------
dict:  
  "time_period":  <describe>
  "frequency": <describe>
  "duty_cycle": <describe>
  "t_low": <describe>
  "t_high": <describe>
```